test/parallel/test-fs-readfile-fd.js

```diff
@@ -79,7 +79,7 @@ function tempFdSync(callback) {
       const buf = Buffer.alloc(5);
 
       // Read only five bytes, so that the position moves to five.
-      fs.read(fd, buf, 0, 5, null, common.mustSucceed((bytes) => {
+      fs.read(fd, buf, 0, 5, -1, common.mustSucceed((bytes) => {
```

Comment:
Can you keep the `null` test please? We want `null` to still works – or mark the PR as `semver-major`.

Reply:
I think this should be `semver-major` as it is supposed to reject a `null` value for `position` which was accepted previously. Thanks for adding the label.

Reply:
On a second thought, we should probably deprecate it before removing it.

Reply:
I added the deprecation warning, test and mentioned the PR link in the YAML. PTAL. :eyes: